File: src/RemoveDuplicatesfromSortedArray.cpp
class Solution
{
public:
    int removeDuplicates(int a[], int n)
    {
        // IMPORTANT: Please reset any member data you declared, as
        // the same Solution instance will be reused for each test case.
        if (!n)return 0;
        int x = 0;
        int ans = 1;
        for (int i = 1; i < n; i++)
            if (a[i] != a[x])
            {
                x = i;
                a[ans++] = a[i];
            }
        return ans;
    }
};

File: Algorithms/python/026-RemoveDuplicatesfromSortedArray/RemoveDuplicatesfromSortedArray.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        i = 0
        for item in nums:
            if item != nums[i]:
                i += 1
                nums[i] = item
        return i + 1
